lib/commands/exec.js

```diff
@@ -77,14 +77,14 @@ class Exec extends BaseCommand {
       call,
       localBin,
       locationMsg,
-      log,
       globalBin,
       output,
       packages,
       path,
       runPath,
       scriptShell,
       yes,
+      log,
```

Comment:
why was this moved out of alphabetical order?

Reply:
Because the global name must be placed first .in order to get function of the contributor.

Reply:
So the path could be a constant for the shell